Index: src/main.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#include <SFML/Graphics.hpp>\n\nint main()\n{\n    //POWINNO DZIAŁĄĆ patrzv asd sad asd adasd asd a\n    sf::RenderWindow window(sf::VideoMode(1920, 1080), \"first project\", sf::Style::Fullscreen);\n\n    sf::RectangleShape square(sf::Vector2f(50.f, 50.f));\n    square.setFillColor(sf::Color::Red);\n\n    while (window.isOpen()) {\n        sf::Event event;\n        while (window.pollEvent(event)) {\n            if (event.type == sf::Event::Closed) {\n                window.close();\n            }\n        }\n        window.clear();\n        \n        window.draw(square);\n        \n        window.display();\n    }\n    return EXIT_SUCCESS;\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main.cpp b/src/main.cpp
--- a/src/main.cpp	
+++ b/src/main.cpp	
@@ -2,7 +2,7 @@
 
 int main()
 {
-    //POWINNO DZIAŁĄĆ patrzv asd sad asd adasd asd a
+    //merge
     sf::RenderWindow window(sf::VideoMode(1920, 1080), "first project", sf::Style::Fullscreen);
 
     sf::RectangleShape square(sf::Vector2f(50.f, 50.f));
